dm/config/task_converters.go

```diff
@@ -308,6 +315,11 @@ func OpenAPITaskToSubTaskConfigs(task *openapi.Task, toDBCfg *dbconfig.DBConfig,
 		if task.IgnoreCheckingItems != nil && len(*task.IgnoreCheckingItems) != 0 {
 			subTaskCfg.IgnoreCheckingItems = *task.IgnoreCheckingItems
 		}
+		// set syncer IO total bytes counter
+		subTaskCfg.IOTotalBytes = atomic.NewUint64(0)
+		subTaskCfg.DumpIOTotalBytes = atomic.NewUint64(0)
+		subTaskCfg.UUID = uuid.NewString()
+		subTaskCfg.DumpUUID = uuid.NewString()
```

Comment:
maybe abstract into a `InitIOCounters` method

Reply:
agree, fixed it